blocks/bricks/sequence_generators.py

```diff
@@ -226,7 +238,7 @@ def generate_states(self):
     @generate.property('outputs')
     def generate_outputs(self):
         return (self.state_names + ['outputs']
-                + self.glimpse_names + ['costs'])
+                + self.glimpse_names + ['costs', 'probs'])
```

Comment:
@rizar Is it going to ruin something? What should I return for emiters which have no support for `probs`?